index.md

```diff
@@ -102,7 +106,9 @@ export default {
 </script>
 ```
 
-## The schema prop
+Keep in mind when using v-model with <SchemaForm>, the value that we pass will be replaced with a new value. This also means we should always use ref to create that state object as it will track the changes as you would expect.
```

Comment:
```suggestion
Keep in mind when using v-model with `<SchemaForm>`, the value that we pass will be replaced with a new value. This also means we should always use `ref` to create that state object as it will track the changes as you would expect.
```

Reply:
oops, good catch